src/createServer.js

```diff
@@ -1,3 +1,62 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const normalizeUrl = new URL(req.url, `https://${req.headers.host}`);
+
+    let response = {};
+    const errors = [];
+
+    const textToConvert = normalizeUrl.pathname.slice(1);
+    const toCase = normalizeUrl.searchParams.get('toCase');
+
+    if (textToConvert.length === 0) {
```

Comment:
will be better to split in separate file logic for errors

Reply:
agree, move this logic to separate function